server/pom.xml

```diff
@@ -744,7 +744,10 @@
       <dependency>
         <groupId>org.zanata</groupId>
         <artifactId>zanata-common-api</artifactId>
+        <!-- release plugin does not play nice with variable substitution
         <version>${zanata.api.compat.version}</version>
+        -->
```

Comment:
I did try with that, it failed with the error message: 
 The artifact (org.zanata:zanata-common-api) requires a different version (4.0.0-alpha-2) than what is found (3.9.1). 

Maybe it is because org.zanata:zanata-common-api is now part of the build (it was in zanata-api instead) and release plugin try to be smart and "correct" it.